gulpfile.js

```diff
@@ -61,14 +62,24 @@ function reloadChromeExtensionsTab(cb) {
     });
 }
 
-function safarify() {
+function babelify() {
     return gulp.src('./build/safari.safariextension/zotero/**/*.js')
         .pipe(buffer())
         .pipe(plumber({errorHandler: onError}))
         .pipe(babel({
             presets: ['es2015']
         }))
-        .pipe(gulp.dest('./build/safari.safariextension/zotero/'));
+        .pipe(gulp.dest((data) => data.base));
+}
+
+function minify() {
+	return gulp.src('./build/**/*.js')
```

Comment:
These lines also don't work properly if a custom build directory is specified in config.sh. That's OK, but then we should remove that config option and always build to ./build.